doc/USER_GUIDE.md

```diff
@@ -0,0 +1,132 @@
+# Installation
```

Comment:
I feel like installation is something near 100% of the users won't need to consider. Perhaps a separate INSTALL.md document?

If you do go for the separate doc, please mention the install-labels script I added recently.

Reply:
Moved to separate doc.